### DISCUSSION TITLE
Ai picture

### DISCUSSION THREAD

**Discussion by LilRaven8** [rocksdanister/lively#2346] [General]
How can I edit my ai picture?

(Sorry, I'm new to lively)

**Comment by LilRaven8** [comment]
Could you add widgets to Lively too? I cant use rainmeater because im in 7th grade using a school comuter

**Comment by rocksdanister** [comment]
> How can I edit my ai picture?

Do you mean depth wallpaper?
https://github.com/rocksdanister/lively/wiki/Machine-Learning

--- -> Open file location.

The files should be inside folder.
